### DISCUSSION TITLE
(OBSOLETE) Temporary 1.19.1/1.19.2 update fix

### DISCUSSION THREAD

**Discussion by Shadowysn** [Draylar/identity#422] [General]
EDIT: Officially fixed. Thanks Draylar!

This is the compiled .jar file of [Alexthw46's fix](https://github.com/Draylar/identity/pull/408) for both **1.19.1** and **1.19.2**. Nothing aside from the command hook crashing has been fixed or changed, it is mainly just the 1.19 mod.
Feel free to run it through VirusTotal. It should be safe.

The **1.19.1/2** version of Architectury API is required. ([Forge](https://www.curseforge.com/minecraft/mc-mods/architectury-api/files/3929407) and [Fabric](https://www.curseforge.com/minecraft/mc-mods/architectury-api/files/3929406) versions links)
[identity-2.5.1-1.19.1-fabric.zip](https://github.com/Draylar/identity/files/9393035/identity-2.5.1-1.19.1-fabric.zip)
[identity-2.5.1-1.19.1-forge.zip](https://github.com/Draylar/identity/files/9393039/identity-2.5.1-1.19.1-forge.zip)

**Comment by zPanic** [comment]
Thanks for posting this!
Do you happen to know if there's a Forge version of this fix?

**Reply by Shadowysn** [reply]
> Thanks for posting this! Do you happen to know if there's a Forge version of this fix?

I heard reports that the forge version is riddled with issues and I was initially not sure if posting the Forge-compiled fix I already have on hand can do much.

**Reply by zPanic** [reply]
Thanks for the quick reply! I'll test it out and otherwise wait for the mod creator to post a new version.

**Reply by Alexthw46** [reply]
We've been using the forge version on a private server for few weeks and it shouldn't have more bugs than 1.19.0 ones.

**Comment by Draylar** [comment]
Hi Shadow, thanks so much for compiling the patched jars for everyone to use! I'll push an official release with it included in a little bit. Feel free to keep the jars up for people to see.

**Reply by zPanic** [reply]
Hi Draylar, thanks for that!
Does this mean the patched jars are confirmed to work with 1.19.2 without major problems?

**Reply by Draylar** [reply]
Just booted up my local jar on a fresh 1.19.2 Fabric instance and it seems to work fine. Let me know if you run into any problems loading it up after I publish it as a release!

![image](https://user-images.githubusercontent.com/17728338/187092300-e403fcb6-48f6-48d7-9994-2907cbd5cf59.png)

**Reply by zPanic** [reply]
No problems so far!
Just that players stay red after taking damage in multiplayer.

**Reply by Alexthw46** [reply]
The two problems i'm aware of are the one that Panic mentioned and was already present in 1.19.0 #424 and the more severe one that causes all the entities to stop render randomly, the only fix for the latter is to close and relaunch the client, saving and re-joining is not enough. Might be an issue with the forge version and/or architectury tho.
